Backend/app/rider-platform/rider-app/Main/src/Beckn/OnDemand/Transformer/OnSearch.hs

```diff
@@ -149,7 +149,7 @@ tfQuotesInfo provider fulfillments validTill item = do
             quoteInfo <- buildRentalQuoteInfo item quoteOrEstId_ currency & Kernel.Utils.Error.fromMaybeM (Tools.Error.InvalidRequest "Missing rental quote details")
             pure $ Domain.Action.Beckn.OnSearch.RentalDetails quoteInfo
           OneWay OneWayRideOtp -> pure $ Domain.Action.Beckn.OnSearch.OneWaySpecialZoneDetails (Domain.Action.Beckn.OnSearch.OneWaySpecialZoneQuoteDetails {quoteId = quoteOrEstId_})
-          InterCity OneWayRideOtp _ -> pure $ Domain.Action.Beckn.OnSearch.OneWaySpecialZoneDetails (Domain.Action.Beckn.OnSearch.OneWaySpecialZoneQuoteDetails {quoteId = quoteOrEstId_})
+          CrossCity OneWayRideOtp _ -> pure $ Domain.Action.Beckn.OnSearch.OneWaySpecialZoneDetails (Domain.Action.Beckn.OnSearch.OneWaySpecialZoneQuoteDetails {quoteId = quoteOrEstId_})
```

Comment:
This InterCity to OneWaySpecialZoneDetails mapping was made in one of my Refactor PR, but now upon checking I had found that @0utkarsh has implemented Ride OTP flow use case in InterCityDetails, so removed that mapping.